bittensor/commands/stake.py

```diff
@@ -614,32 +616,43 @@ def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         if not cli.config.is_set("netuid"):
             cli.config.netuid = int(Prompt.ask("Enter netuid"))
 
+        netuid = cli.config.netuid
+        total_subnets = subtensor.get_total_subnets()
+        if total_subnets is not None and total_subnets <= netuid <= 0:
```

Comment:
Do we mean this instead?
`if total_subnets is not None and 0 <= netuid < total_subnets:`

Reply:
The opposite
If total subnets is not None, and netuid is out of range, then raise error